### PR TITLE
DS-506 v3 Docs

### PR DESCRIPTION
**Pull request by froboy**
- [DS-506](https://yusa.atlassian.net/browse/DS-506) - Lb v3 docs

### CODE REVIEW

File: content/en/docs/user-documentation/layout-builder/donate/_index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Troubleshooting
+
+If your embedded form does not work in your non-production environment you may need to add a domain to the allow-list either on the provider-side or in your site's Content Security Policy.
+
+If your provider is not listed you can add the form by selecting the **Code** Custom Block and then pasting in your code. Alternatively you can work with your development partner to [add a new donation provider](https://git.drupalcode.org/project/y_donate#information-for-developers).
+
+## Donate Block
+
+The Donate Block can be placed in a full-width container.
```
Comment (podarok) [inline content/en/docs/user-documentation/layout-builder/donate/_index.md:36]: Let's explain or link what|how to place in `full-width` container

File: content/en/docs/user-documentation/layout-builder/location-finder/_index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### Hierarchical (parent/child) amenities
+
+![A screenshot showing a parent-child relationship in the Location finder filters.](lb-location-finder--parent-amenities.png)
+
+Y's with many amenities may choose to group them in categories. Once any Amenities term is nested, the Location Finder filters switch to a hierarchical display.
+
+To nest terms:
+
+- Go to the Amenities administration page.
```
Comment (podarok) [inline content/en/docs/user-documentation/layout-builder/location-finder/_index.md:42]: Can we have a link?

**Review by podarok — APPROVED** [review]

File: content/en/docs/user-documentation/layout-builder/donate/_index.md (hunk 1 above)
Comment (froboy) [reply content/en/docs/user-documentation/layout-builder/donate/_index.md:36]: Fixed in https://github.com/YCloudYUSA/yusaopeny_docs/commit/2816b0248fd0c0a1c968a718dbe56ed40881f4a5

File: content/en/docs/user-documentation/layout-builder/location-finder/_index.md (hunk 2 above)
Comment (froboy) [reply content/en/docs/user-documentation/layout-builder/location-finder/_index.md:42]: Fixed in https://github.com/YCloudYUSA/yusaopeny_docs/commit/b608cd335f154ab57cb96b75331e91d61b46013b

**Pull request merged**